rolling/release-rhel-build.yaml

```
@@ -30,6 +30,7 @@ package_blacklist:
 - joint_state_broadcaster  # Build failures on RHEL
 - microstrain_inertial_msgs  # Not yet generated for RHEL
 - moveit_common  # libogre-dev has no RPM package for RHEL
+- moveit_resources_prbt_support  # Not yet generated for RHEL
```

The reason for the missing Bloom branches is a rosdep resolution failure.
```suggestion
- moveit_resources_prbt_support  # libogre-dev has no RPM package for RHEL
```